src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package step2;
+
+public class Calculator {
+    public int sum(int num1, int num2) {
+
+        return num1+num2;
```

Comment:
코드 컨벤션 한번 확인 부탁 드립니다 😄 
```suggestion
        return num1 + num2;
```